### PR TITLE
doc: update octopus release notes for rgw

### PR DESCRIPTION
**Pull request by cbodley**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by cbodley** [comment]
@yuvalif i know we've backported a lot of your notification work; anything you'd like to mention for the Octopus release?

**Comment by cbodley** [comment]
jenkins render docs

**Comment by cbodley** [comment]
jenkins render docs

File: doc/releases/octopus.rst [hunk 1]
```diff
@@ -124,9 +124,18 @@ RADOS
...
   
-* Multi-site replication can now be managed on a per-bucket basis (EXPERIMENTAL).
-* WORM?
-* bucket tagging?
+* New `Multisite Sync Policy`_ primitives for per-bucket replication. (EXPERIMENTAL)
+* S3 feature support:
+    - Bucket Replication (EXPERIMENTAL)
+    - Bucket Tagging
+    - Object Lock
+    - Public Access Block
```
Comment (theanalyst) [inline doc/releases/octopus.rst]: maybe worth mentioning `for buckets`

Comment (theanalyst) [reply doc/releases/octopus.rst]: List Objects V2

Comment (theanalyst) [reply doc/releases/octopus.rst]: nvm list v2 was already backported in nautilus

Comment (cbodley) [reply doc/releases/octopus.rst]: > maybe worth mentioning `for buckets`

added thanks

File: doc/releases/octopus.rst [hunk 2]
```diff
@@ -124,9 +124,18 @@ RADOS
...
+* New `Multisite Sync Policy`_ primitives for per-bucket replication. (EXPERIMENTAL)
+* S3 feature support:
+    - Bucket Replication (EXPERIMENTAL)
+    - Bucket Tagging
+    - Object Lock
+    - Public Access Block for buckets
+* Bucket sharding:
+    - Significantly improved listing performance on buckets with many shards.
+    - Dynamic resharding prefers prime shard counts for improved distribution.
+    - Raised the default number of bucket shards to 11.
```
Comment (theanalyst) [inline doc/releases/octopus.rst]: maybe worth adding the content in PendingReleaseNotes about this as we've added in https://github.com/ceph/ceph/pull/32660 about the zone params

**Comment by yuvalif** [comment]
> @yuvalif i know we've backported a lot of your notification work; anything you'd like to mention for the Octopus release?

@cbodley there is some work that was not backported yet, but should be in the next nautilus release. do we need to mention it here?

**Comment by yuvalif** [comment]
suggested bucket notification release notes:
```
This version adds Bucket Notification on top of the existing PubSub sync module, as a mechanism for sharing bucket changes with external system. 
The notifications are not stored inside Ceph, and are pushed directly to the configured endpoints.
This allows for more fine grained granularity of filtering on event types, as well as richer set of information to be shared.
Supported endpoints types are: HTTP/S, AMQP0.9.1 and Kafka.
In this version events could be filtered based on:
- the object's suffix/prefix/regex
- the object's metadata (which is also sent in the notification)
- the object's tags (which are also sent in the notification)
Note that using bucket notifications does not require a dedicated "pubsub" zone.
```

File: doc/releases/octopus.rst (hunk 2 above)
Comment (cbodley) [reply doc/releases/octopus.rst]: they'll already see that stuff in release notes

**Comment by cbodley** [comment]
thanks @yuvalif - i think that's too much detail here, so i just linked to the notifications.rst doc

**Pull request merged**